operator/src/main/resources/Operator.properties

```diff
@@ -157,6 +157,14 @@ WLSKO-0201=WL pod shutdown: Failed to shutdown WebLogic server {0} via REST inte
 WLSKO-0202=WL pod shutdown: Failed to shutdown WebLogic server {0} via REST interface due to HTTP request timeout after {1} seconds.
 WLSKO-0203=WL pod shutdown: Failed to shutdown WebLogic server {0} via REST interface due to exception: {1}.
 WLSKO-0204=WL pod shutdown: Retry shutdown of WebLogic server {0} via REST interface.
+WLSKO-0205=Conversion Webhook For Oracle WebLogic Kubernetes Operator, version: {0}, implementation: {1}, build time: {2}
```

Comment:
I wonder if give the webhook a precise name in all public references. This not only future proofs the feature in case additional webhooks are added later, but it helps make its purpose more clear, and sets a consistent name for documentation purposes. If so, then using "WebLogic Conversion Webhook" (WCW) or "Domain Conversion Webhook" (DCW) (or similar) might be adequate, and should be used in place of just plain 'webhook' in the other log messages and anywhere else that is public.

Reply:
Since the conversion webhook is used only in the context of the WebLogic Operator, how about "WebLogic Operator Conversion Webhook" or "WKO Conversion Webhook"? I'm fine with other names as well. @rjeberhard ?

Reply:
Kubernetes calls this a "custom resource conversion webhook." The full name could be something like, "WebLogic Domain Custom Resource Conversion Webhook" and then in the rest of the document we can just call it the "webhook" or "conversion webhook."

Reply:
Changed the message to use "WebLogic Domain custom resource conversion webhook".